### ISSUE TITLE
Ankama Launcher does not work

### ISSUE THREAD

**Issue by LeandroStanger** [RobLoach/net.lutris.Lutris#42]
```
Running /sysroot/home/leandro/Games/ankama-launcher/AnkamaLauncher64.AppImage
fuse: failed to exec fusermount: No such file or directory

Cannot mount AppImage, please check your FUSE setup.
You might still be able to extract the contents of this AppImage 
if you run it with the --appimage-extract option. 
See https://github.com/AppImage/AppImageKit/wiki/FUSE 
for more information
open dir error: No such file or directory
Waiting on children
All children gone
Exit with returncode 1
```

**Comment by gasinvein** [comment]
AppImage relies on fuse, which won't work in flatpak sandbox. You can extract the appimage is as it suggests and try running extracted binary.

**Comment by lionirdeadman** [comment]
Possibly could talk with Lutris team to extract it and run the executable if the system can't create fuses? @gasinvein

**Comment by gasinvein** [comment]
How Lutris and this Ankama Launcher are related btw? I can't find anything in Lutris code mentioning it.

**Comment by RobLoach** [comment]
Is this https://lutris.net/games/ankama-launcher/ ? In order to run AppImages, would we have to ship fuse as part of the Flatpak?

**Comment by gasinvein** [comment]
@RobLoach 
> https://lutris.net/games/ankama-launcher/ 

This looks like something done by third-party, not by Lutris devs.

>In order to run AppImages, would we have to ship fuse as part of the Flatpak?

It won't be enough, I guess. Does bwrap sandbox even allow fuse interaction?

**Comment by refi64** [comment]
AFAIK FUSE isn't supported inside of sandboxes at all due to kernel constraints. However, you could extract the AppImage, using either `--appimage-extract` or [unappimage](https://github.com/refi64/unappimage) (which is used by the Unity Hub Flatpak, since `--appimage-extract` wants access to /proc which isn't available in apply_extra).

**Comment by gasinvein** [comment]
@refi64, true, thus this can't be fixed on our side. I think the best we can is to to ask the author of [this](https://lutris.net/games/ankama-launcher/) to extract the appimage instead of running it as is.

**Comment by LeandroStanger** [comment]
The RPCS3 is also a AppImage application https://github.com/RPCS3/rpcs3/issues/5935

**Comment by gasinvein** [comment]
RPCS3 is a runner in Lutris terms, not a 3rd-party game like Ankama Launcher. And we (well, at least me) plan to build runners ourselves instead of poking around prebuilt binaries.

**Comment by qurious-pixel** [comment]
`--appimage-extract-and-run` might be the command you want. This temporarily extracts the appimage, and then runs the binary, and once it closes, cleans up the temp folder. This works without fuse.
